drivers/ieee802154/ieee802154_kw41z.c

```diff
@@ -82,6 +221,7 @@ struct kw41z_context {
 
 static struct kw41z_context kw41z_context_data;
 
+
```

Comment:
lets remove the blank line.

Reply:
Lot of "blank line" comments. Is there a way to get this added to the checkpath.pl script?